rules-reviewed/cloud/java-rmi.windup.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<ruleset id="java-rmi" xmlns="http://windup.jboss.org/schema/jboss-ruleset" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This is a ruleset for Java Remote Method Invocation specific rules while migrating to Cloud environment.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.7.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.7.0.Final" />
+        </dependencies>
+        <sourceTechnology id="java" />
+        <sourceTechnology id="rmi" />
+        <targetTechnology id="cloud" />
```

Comment:
Really, I would prefer "openshift" or some other product name. The "cloud" will have different rules depending upon which cloud offering is used.